lib/mayaUsd/render/vp2ShaderFragments/usdPreviewSurfaceLightingAPI1.xml

```diff
@@ -185,7 +185,7 @@ usdPreviewSurfaceLightingAPI1(
         if (!useSpecularWorkflow) {
             float R = (1.0 - ior) / (1.0 + ior);
             float3 specColor = mix(float3(1.0, 1.0, 1.0), diffuseColor, metallic);
-            F0  = R * R * specColor;
+            F0 = mix(R * R * specColor, specColor, metallic);
```

Comment:
Fixes issue 3 by using the formula from [USD](https://github.com/PixarAnimationStudios/USD/commit/f11ab360)